README.md

```diff
@@ -1,9 +1,11 @@
 # vpc-router
 
+vpc-router lets users users avoid route table limitations and build large Kubernetes clusters with the performance and visability of native VPC networking.
+
 ## Introduction
 
-vpc-router is a utility for the setting/deleting of routes in a VPC route
-table, consisting of a destination CIDR as well as the IP address of an EC2
+vpc-router is a utility for setting and deleting routes in an Amazon EC2 VPC route
+table. Each route is specified by a destination CIDR as well as the IP address of an EC2
```

Comment:
Actually, it has to be "route tables" (plural), since it really will just set the routes in all the route tables it finds in a VPC and there are usually at least two.